controllers/userController.js

```diff
@@ -2,7 +2,7 @@ const config = require("../config/config");
 const users = require("../model/userModel");
 const bcrypt = require("bcryptjs");
 const jwt = require("jsonwebtoken");
-const {updateuser1,getdata,matchpass,modifyPass,verifyemail,userlogin,usersignup,user_list} = require("../services/userservices")
+const {updateuser1,getdata,useraddress,matchpass,modifyPass,verifyemail,userlogin,usersignup,user_list} = require("../services/userservices")
```

Comment:
use classes or object and keep everything inside it a lot of import like this doesn't look good